examples/notebooks/robust_models_1.ipynb

```diff
@@ -263,7 +263,7 @@
     "c = 4.685\n",
     "support = np.linspace(-3*c, 3*c, 1000)\n",
     "tukey = norms.TukeyBiweight(c=c)\n",
-    "plot_weights(support, tukey.weights, ['-3*c', '0', '3*c'], [-3*c, 0, 3*c]);"
```

Comment:
those semicolons are INTENTIONAL.
they are to prevent the plots from being rendered twice

Reply:
How does that work?  How would matplotlib even know there's a semicolon?

Reply:
It's not matplotlib, it's jupyter
https://github.com/statsmodels/statsmodels/issues/1265#issuecomment-383336489

Reply:
Well that's a kick in the teeth.

I think the thing to do here is to close this PR and put together a robust long-term git-hook-resembling solution that keeps the semicolon in the .ipynb version and removes it in the .py version.  (and to encourage you to merge #5036)